packages/bot/src/components/journal.jsx

```diff
@@ -0,0 +1,125 @@
+import classnames     from 'classnames';
+import proptypes      from 'prop-types';
+import React          from 'react';
+import { Scrollbars } from 'tt-react-custom-scrollbars';
+import { connect }    from '../stores/connect';
+import { translate }  from '../utils/tools';
+import '../assets/sass/journal.scss';
+
+const DateItem = ({
+    date, time,
+}) => {
+    return (
+        <div>
+            <span className='journal__table--date'> {date} </span>
+            <span className='journal__table--time'> {time} </span>
+        </div>
+    );
+};
+
+const FormatMessage = ({
+    message,
+}) => {
+    const keyWords = ['Bought', 'Sold' , 'Profit amount', 'Loss amount'];
+    const messages = message.split(':');
+    if (messages.count < 2) {
+        return message;
+    }
+
+    let title_color,
+        value_color;
+    
+    switch (messages[0]) {
+        case (keyWords[0]) :{
```

Comment:
Maybe we can switch on the name directly here, it would make the code more clear.

Reply:
@aaron-binary its better using constants everywhere we need to hardcode.